assets/css/style.css

```diff
@@ -0,0 +1,288 @@
+* {
+  /* make widths static */
+  box-sizing: border-box;
+}
+
+body {
+  background: #eee;
+  /* Image options for backgrounds:
+  background-image: url("../img/dot-grid.png");
+  background-image: url("../img/grey.png"); */
+  background-image: url("../img/grey_@2X.png");
+  color: #777;
+  font: 16px/1.5em 'Open Sans', sans-serif;
+}
+
+/******************************************************************************
+ Header Elements
+*******************************************************************************/
+header {
+  letter-spacing: 3px;
+}
+
+header, 
+.main-container {
+  /* DEBUG:
+     border: 1px red solid; */ 
+  margin: 0 auto;
+  width: 960px;
+}
+
+header,
+h1 {
+  font-family: 'Raleway', sans-serif;
+}
+
+/******************************************************************************
+ Navigation Elements
+*******************************************************************************/
+nav {
+  float: right;
+  width: 30%;
+}
+
+nav ul {
+  float: right;
+  list-style-type: none;
+  margin: 2em 0 0 0;
+  overflow: hidden;
+  padding: 0;
+}
+
+nav ul li {
+  border-right: 1px solid #ccc;
+  float: left;
+}
+
+nav ul li:last-child {
+  border-right: none;
+}
+
+nav ul li a {
+  color: #777;
+  display: block;
+  font-size: 0.8em;
+  line-height: 1em;
+  padding: 3px 10px;
+  text-align: center;
+}
+
+nav ul li a:active,
+nav ul li a:hover {
+  /* Underline is more aestethically pleasing here. */
+  /* background: #ddd; */
+}
+
+nav ul li a:visited {
+  color: #aaa;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:active,
+a:hover {
+  text-decoration: underline;
+}
+
+/******************************************************************************
+ Main Content Elements
+*******************************************************************************/
+main {
+  background: #fff;
+  border: 1px solid #ddd;
+  padding: 0 20px;
+}
+
+h1,
+main a {
+  color: #4aaaa5;
+}
+
+h1 {
+  border-bottom: 3px #ddd solid;
+  padding: 10px 0;
+}
+
+main img {
+  /* ISSUE: only rounding top, left corner (20200215). */
+  /* border-radius: 20px; */
+  float: left;
+  padding: 0 20px 20px 0;
+  width: 240px;
+}
+
+main p {
+  font-size: 20px;
+  line-height: 2em;
+}
+
+main figure img {
+  padding: 0;
+}
+
+main figure img,
+.photo-label {
+  width: 240px;
+}
+
+/******************************************************************************
+ Form Elements
+*******************************************************************************/
+main form input[type=text],
+main form input[type=email],
+main form textarea {
+  color: #ccc;
+  padding: 5px 10px;
+  width: 100%;
+}
+
+/* Form input "glow" compliments of CSS Tricks:
+   https://css-tricks.com/snippets/css/glowing-blue-input-highlights/ */
+main form input,
+main form textarea {
+  -webkit-transition: all 0.30s ease-in-out;
+  -moz-transition: all 0.30s ease-in-out;
+  -ms-transition: all 0.30s ease-in-out;
+  -o-transition: all 0.30s ease-in-out;
```

Comment:
Nice! FYI there are tools that do this vendor-prefixing for you. However, I _think_ you don't need the vendor prefixes any more on this one. I think you can just use one `transition: ...`

[<img width="1288" alt="image" src="https://user-images.githubusercontent.com/661038/74705697-a06d1080-51c9-11ea-955a-60eb8fb7ed41.png">](https://caniuse.com/#search=transition)